packages/examples/security/oauth/README.md

```diff
@@ -0,0 +1,36 @@
+# oAuth example
+The example is composed by two actors: one Thing that expose an action with oAuth2.0 client credential security constraint and a client who want to use that function. In the example, we are using an utility server to generate tokens and validate client credentials. Therefore, the **wot-consumer** force the action `href` to be the same as the utility server with the goal to validate the obtained oAuth2.0 token. In the feature the exposing servient could also play this role. 
+
+## run the example
+
+```bash
+npm install
+npm run build
+```
+After this follow the procedure described in [here](../../README.md) and remove the unwanted line of codes inside the scripts `./dist/consumer.js` and `./dist/exposer.js`.
+
+Now you are ready to run the example.
+
+```bash
+# start the server
+npm run server
+```
+in a different terminal 
+
+```bash
+# start the exposer
+npm run start:exposer
+```
+Finally, in other terminal
+```bash
+npm run start:consumer
```

Comment:
I get the following

```
...
ConsumedThing 'OAuth' setting credentials for [HttpClient]
Servient looking up credentials for 'urn:dev:wot:oauth:test'
HttpClient using security scheme 'oauth2'
ConsumedThing 'OAuth' invoking https://localhost:3000/resource
LOCALHOST FIX
HttpClient (invokeResource) sending GET  to https://127.0.0.1:3000/resource
(node:38356) UnhandledPromiseRejectionWarning: FetchError: request to https://127.0.0.1:3000/resource failed, reason: certificate has expired
    at ClientRequest.<anonymous> (C:\Projects\WoT\thingweb\thingweb.node-wot\packages\binding-http\node_modules\node-fetch\lib\index.js:1455:11)
```

Reply:
The issue is that 

```
    "http":{
        "allowSelfSigned": true
    },
```

is missing in
examples\security\oauth\wot-client-servient-conf.json
different to 
packages\examples\security\oauth\wot-client-servient-conf.json

With this addition it works in both places.

May I ask you to add it to the PR...

Reply:
Nice catch! solved 👍